sale_order_product_recommendation_packaging_default/tests/test_recommendation.py

```diff
@@ -188,6 +188,33 @@ def test_preexisting_product(self):
             ],
         )
 
+    def test_preexisting_product_without_modification(self):
+        self.new_so.order_line = [
+            Command.create(
+                {
+                    "product_id": self.prod_1.id,
+                    "product_uom_qty": 12,
+                    "qty_delivered_method": "manual",
+                    "discount": 12.0,
+                }
+            )
+        ]
+        # Do not modify anything
+        wiz = self.wizard()
+        wiz.action_accept()
+        self.assertRecordValues(
+            self.new_so.order_line,
+            [
+                {
+                    "product_id": self.prod_1.id,
+                    "product_packaging_id": self.prod_1_dozen.id,
+                    "product_packaging_qty": 1,
+                    "product_uom_qty": 12,
+                    "discount": 12.0,  # Discount is preserved
```

Comment:
Are you sure this test is actually working? If you don't set a pricelist, I think it would be preserved anyway.

Reply:
Default pricelist is always applied. 

Every time a pricelist with Discount Policy "Show discount and price" recomputes something, discounts are reset.
If the Discount Policy on the pricelist is "Discount included in price", preserves used discounts.